File: services/kpi-service/Hinet.Service/Constant/KyQuyConstant.cs
using System.ComponentModel;

namespace Hinet.Service.Constant
{
    public class KyQuyConstant
    {
        [DisplayName("Tạm lưu")]                  public static int TamLuu => 0;
        [DisplayName("Chờ duyệt")]                public static int ChoDuyet => 1;
        [DisplayName("Đề nghị chỉnh sửa")]        public static int DeNghiChinhSua => 2;
        [DisplayName("Bị từ chối")]               public static int BiTuChoi => 3;
        [DisplayName("Đã duyệt điện tử")]         public static int DaDuyetDienTu => 4;
        [DisplayName("Đã xác nhận")]              public static int DaXacNhan => 5;
        [DisplayName("Cần bổ sung thông tin")]    public static int CanBoSungThongTin => 6;
        [DisplayName("Đã huỷ đăng ký")]           public static int DaHuyDangKy => 8;
        [DisplayName("Đã review")]                public static int DaReview => 26;
        [DisplayName("Cần bản giấy")]             public static int CanBanGiay => 28;
    }
}

File: clients/asset-client/src/services/rutTienKyQuy/KyQuyConstant.ts
export const KyQuyConstant = {
  TamLuu: 0,
  ChoDuyet: 1,
  DeNghiChinhSua: 2,
  BiTuChoi: 3,
  DaDuyetDienTu: 4,
  DaXacNhan: 5,
  CanBoSungThongTin: 6,
  DaHuyDangKy: 8,
  DaReview: 26,
  CanBanGiay: 28,
};

export const KyQuyStatusNames: { [key: number]: string } = {
  [KyQuyConstant.TamLuu]: "Tạm lưu",
  [KyQuyConstant.ChoDuyet]: "Chờ duyệt",
  [KyQuyConstant.DeNghiChinhSua]: "Đề nghị chỉnh sửa",
  [KyQuyConstant.BiTuChoi]: "Bị từ chối",
  [KyQuyConstant.DaDuyetDienTu]: "Đã duyệt điện tử",
  [KyQuyConstant.DaXacNhan]: "Đã xác nhận",
  [KyQuyConstant.CanBoSungThongTin]: "Cần bổ sung thông tin",
  [KyQuyConstant.DaHuyDangKy]: "Đã huỷ đăng ký",
  [KyQuyConstant.DaReview]: "Đã review (Trình Lãnh đạo)",
  [KyQuyConstant.CanBanGiay]: "Cần bản giấy",
};

export const KyQuyStatusColors: { [key: number]: string } = {
  [KyQuyConstant.TamLuu]: "default",
  [KyQuyConstant.ChoDuyet]: "processing",
  [KyQuyConstant.DeNghiChinhSua]: "warning",
  [KyQuyConstant.BiTuChoi]: "error",
  [KyQuyConstant.DaDuyetDienTu]: "cyan",
  [KyQuyConstant.DaXacNhan]: "success",
  [KyQuyConstant.CanBoSungThongTin]: "orange",
  [KyQuyConstant.DaHuyDangKy]: "purple",
  [KyQuyConstant.DaReview]: "blue",
  [KyQuyConstant.CanBanGiay]: "geekblue",
};
